sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokers.java

```diff
@@ -397,7 +397,7 @@ public boolean isValid() {
     private Object describeType(Type type) {
```

Comment:
I think there is one other place we use the descriptor (for the return value in locals). Should that also be getInternalName() and/or use describeType?